modules/ROOT/pages/api-groups-landing-page.adoc

```diff
@@ -0,0 +1,69 @@
+= API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+API Groups enables you to bundle your APIs and specific resources to solve specific user needs. Instead of individually using a list of resources and APIs, the API users can request access to and use these APIs as a package that solves a specific problem for them. 
```

Comment:
```suggestion
Using API Groups enables you to bundle your APIs and resources to solve specific user needs. Instead of using individual resources and APIs from a list, the users can request access to and use these in a package that solves a specific problem for them. 
```

Reply:
Done! Thanks!